Source/WhisperFactory.swift

```diff
@@ -187,6 +216,20 @@ class WhisperFactory: NSObject {
 
 extension WhisperFactory: UINavigationControllerDelegate {
```

Comment:
@zenangst @RamonGilabert With the name like `WhisperFactory` I'm expecting to see creation of objects, but it's not the purpose of this class. Might be only me.